### PR TITLE
777 leaf777 patch 1

### PR DESCRIPTION
**Pull request by 777Leaf777**
Fixed the Floating points aad changed the map encoding. might need to delete the Tabba_town.tmx and replace mine with ZhongTiaos more up to date version.

### CODE REVIEW

File: Azure_Town.tmx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  <object id="3" name="Also Tabathas House" type="event" x="130" y="160" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport tabathas.tmx,6,9"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing up"/>
+   </properties>
+  </object>
+  <object id="5" name="Read Sign" type="event" x="640" y="368" width="16" height="16">
+   <properties>
+    <property name="act10" value="dialog  Town Hall"/>
```
Comment (bitcraft) [inline Azure_Town.tmx:74]: Use translated_dialog unless it cannot be used for some reason.

File: mods/tuxemon/maps/professor_lab.tmx [hunk 2]
```diff
@@ -53,13 +64,66 @@
     <property name="cond20" value="is player_facing down"/>
    </properties>
   </object>
-  <object id="69" name="Choose Creature 1" x="192" y="96" width="16" height="16"/>
-  <object id="74" name="Choose Creature 2" x="208" y="96" width="16" height="16"/>
-  <object id="75" name="Choose Creature 3" x="224" y="96" width="16" height="16"/>
+  <object id="69" name="Choose Rockitten" type="event" x="192.091" y="112.035" width="16" height="16">
+   <properties>
+    <property name="act10" value="add_monster rockitten,5"/>
+    <property name="act20" value="dialog you picked RockKitten"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/professor_lab.tmx:70]: FYI, this is Rockitten, not RockKitten
https://wiki.tuxemon.org/index.php?title=Rockitten

File: mods/tuxemon/maps/taba_town.tmx [hunk 3]
```diff
@@ -1,835 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bitcraft) [inline mods/tuxemon/maps/taba_town.tmx:1]: why is taba_town deleted?

File: mods/tuxemon/maps/timber_town.tmx [hunk 4]
```diff
@@ -155,10 +155,25 @@
...
+   </properties>
+  </object>
+  <object id="189" name="To_Route_7" type="event" x="464.46" y="623.352" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport witcher_route_7.tmx,8,3"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing down"/>
+   </properties>
+  </object>
+  <object id="190" x="404.801" y="388.55"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/timber_town.tmx:172]: float

File: mods/tuxemon/maps/timber_town.tmx [hunk 5]
```diff
@@ -155,10 +155,25 @@
...
  </objectgroup>
  <objectgroup color="#ff0000" id="6" name="Collision">
   <object id="119" type="collision" x="0" y="0" width="32" height="464"/>
-  <object id="120" type="collision" x="0" y="608" width="640" height="32"/>
+  <object id="120" type="collision" x="0" y="608" width="447.202" height="32"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/timber_town.tmx:176]: float

File: mods/tuxemon/maps/witcher_route_7.tmx [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+233,234,170,170,170,170,0,0,0,0,0,0,0,0,0,170,170,170,170,0,
+265,266,170,170,170,170,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+233,234,170,170,170,170,465,0,0,0,0,0,0,0,0,0,0,0,0,0,
+265,266,170,170,170,170,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+233,234,233,234,233,234,0,0,0,465,233,234,233,234,233,234,233,234,0,0,
+265,266,265,266,265,266,465,465,465,465,265,266,265,266,265,266,265,266,0,0
+</data>
+ </layer>
+ <objectgroup color="#ff0000" id="7" name="Collisions">
+  <object id="14" type="collision" x="-7.47922" y="227.424" width="36.2881" height="252.078"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/witcher_route_7.tmx:87]: float

File: mods/tuxemon/maps/witcher_route_7.tmx [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+   </properties>
+  </object>
+  <object id="46" name="To Azure Town" type="event" x="0.277004" y="194.413" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport Azure_Town.tmx,48,28"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing left"/>
+   </properties>
+  </object>
+  <object id="47" x="41.8283" y="234.025" height="0.277008"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/witcher_route_7.tmx:202]: float

File: timber_town.tmx [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+   </properties>
+  </object>
+  <object id="189" name="To_Route_7" type="event" x="464.46" y="623.352" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport witcher_route_7.tmx,8,3"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing down"/>
+   </properties>
+  </object>
+  <object id="190" x="404.801" y="388.55"/>
```
Comment (bitcraft) [inline timber_town.tmx:172]: float

File: witcher_route_7.tmx [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+ <layer id="3" name="Tile Layer 2" width="20" height="30">
+  <data encoding="base64" compression="zlib">
+   eJxjYGBgeAnEr7DQ2MAjTgT7CSd2NZyMDAxcjJg0Oebhcws55uFzCyHz7kLZ95DEVuHBhMzDBqhp3lVO0s17BdTzGgd+Q4Z5vEDMhwPzk+FfXdzeZdAfgeaRCuhtHqn5l5B5pOZfarvPF4j9cGB/MtwHAmZMuOVIdR8IeOExjxz34QPkuA8foMR92PIRJe7Dlj+p4T5k82DuI6YsIMV95JiHDeBy30Uyw4BW7sPXdgG5FZc6dPW42i4X0TAudejpAwD+dHkx
+  </data>
+ </layer>
+ <objectgroup color="#ff0000" id="7" name="Collisions">
+  <object id="14" type="collision" x="0" y="224" width="36" height="252"/>
+  <object id="16" type="collision" x="0" y="448" width="323" height="39"/>
+  <object id="20" type="collision" x="304" y="288" width="22" height="171"/>
+  <object id="23" type="collision" x="160" y="0" width="36.0111" height="123.269"/>
```
Comment (bitcraft) [inline witcher_route_7.tmx:32]: float

File: mods/tuxemon/maps/Azure_Town.tmx [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+  <object id="3" name="Also Tabathas House" type="event" x="130" y="160" width="16" height="16">
+   <properties>
+    <property name="act10" value="teleport tabathas.tmx,6,9"/>
+    <property name="cond10" value="is player_at"/>
+    <property name="cond20" value="is player_facing up"/>
+   </properties>
+  </object>
+  <object id="5" name="Read Sign" type="event" x="640.75" y="368.75" width="16" height="16">
+   <properties>
+    <property name="act10" value="dialog  Town Hall"/>
```
Comment (bitcraft) [inline mods/tuxemon/maps/Azure_Town.tmx:266]: use translated dialog

**Review by bitcraft — CHANGES_REQUESTED** [review]

**Comment by xirsoi** [comment]
Closing due to abandonment

**Pull request closed without merging**